app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user.present?
+
+    flash[:danger] = t "user.show.not_found_user"
+    redirect_to controller: "users", action: "new"
```

Comment:
```suggestion
    redirect_to :new
```
Như này không được à em

Reply:
dạ được anh